js/spec/index-spec.js

```diff
@@ -684,6 +709,12 @@ describe('Node Sentinel File Watcher', function() {
   describe('Garbage collection', function() {
     it('can garbage collect all instances', async function () {
       this.timeout(60000);
+      try {
+        await nsfw(workDir, () => {}, { errorCallback: 'not a callback' });
+        assert.fail('nsfw must throw');
+      } catch (e) {
+        //
+      }
```

Comment:
If you follow this code - if nsfw successfully builds, line 714 will throw an assertion failure which will be swallowed by the catch block.

```javascript
let threw = false;
try {
  await nsfw(workDir, () => {}, { errorCallback: 'not a callback' });
} catch (e) {
  threw = true;
}

if (!threw) {
  assert.fail('nsfw must throw if provided a bad callback');
}
```

Reply:
Good catch, I often fall for this mistake with try blocks.